scripts/run_lighthouse_accessibility_tests.py

```diff
@@ -0,0 +1,188 @@
+    """Deactivates webpages and deletes html lighthouse reports."""
+
+    pattern = 'CONTRIBUTOR_DASHBOARD_ENABLED = .*'
+    replace = 'CONTRIBUTOR_DASHBOARD_ENABLED = False'
+    common.inplace_replace_file(FECONF_FILE_PATH, pattern, replace)
+
+    pattern = '"ENABLE_ACCOUNT_DELETION": .*'
+    replace = '"ENABLE_ACCOUNT_DELETION": false,'
+    common.inplace_replace_file(CONSTANTS_FILE_PATH, pattern, replace)
+
+    build.set_constants_to_default()
+
+    google_app_engine_path = '%s/' % common.GOOGLE_APP_ENGINE_SDK_HOME
+    processes_to_kill = [
+        '.*%s.*' % re.escape(google_app_engine_path),
+    ]
+    for p in SUBPROCESSES:
+        p.kill()
+
+    for p in processes_to_kill:
+        common.kill_processes_based_on_regex(p)
+
+    common.stop_redis_server()
+
+
+def run_lighthouse_puppeteer_script():
+    """Runs puppeteer script to collect dynamic urls."""
+    puppeteer_path = os.path.join(
+        'core', 'tests', 'puppeteer', 'lighthouse_setup.js')
+    bash_command = [common.NODE_BIN_PATH, puppeteer_path]
+
+    try:
+        script_output = subprocess.check_output(bash_command).split('\n')
+        python_utils.PRINT(script_output)
+        for url in script_output:
+            export_url(url)
+        python_utils.PRINT(
+            'Puppeteer script completed successfully.')
+
+    except subprocess.CalledProcessError:
+        python_utils.PRINT(
+            'Puppeteer script failed. More details can be found above.')
+        sys.exit(1)
+
+
+def export_url(url):
+    """Exports the url to an environmental variable."""
+    url_list = url.split('/')
+    if 'collection_editor' in url:
+        os.environ['collection_editor'] = url_list[5]
+    elif 'create' in url:
+        os.environ['exploration_editor'] = url_list[4]
+    elif 'topic_editor' in url:
+        os.environ['topic_editor'] = url_list[4]
+    elif 'story_editor' in url:
+        os.environ['story_editor'] = url_list[4]
+    elif 'skill_editor' in url:
+        os.environ['skill_editor'] = url_list[4]
+
+
+def run_webpack_compilation(source_maps=False):
+    """Runs webpack compilation."""
+    max_tries = 5
+    webpack_bundles_dir_name = 'webpack_bundles'
+    for _ in python_utils.RANGE(max_tries):
+        try:
+            webpack_config_file = (
+                build.WEBPACK_DEV_SOURCE_MAPS_CONFIG if source_maps
+                else build.WEBPACK_DEV_CONFIG)
+            subprocess.check_call([
+                common.NODE_BIN_PATH, WEBPACK_BIN_PATH, '--config',
+                webpack_config_file])
+        except subprocess.CalledProcessError as error:
+            python_utils.PRINT(error.output)
+            sys.exit(error.returncode)
+            return
+        if os.path.isdir(webpack_bundles_dir_name):
+            break
+    if not os.path.isdir(webpack_bundles_dir_name):
+        python_utils.PRINT(
+            'Failed to complete webpack compilation, exiting ...')
+        sys.exit(1)
+
+
+def run_lighthouse_checks():
```

Comment:
run_lighthouse_accessibility_checks* (Considering we have `run_lighthouse_checks` method in another test script as well.)

Reply:
This is outdated now that the lighthouse scripts have been combined as described in this [comment](https://github.com/oppia/oppia/pull/10571/files/ee7b2cd76163db773b69d3ef0a4257b91d207f4f#r484164387)